pkgs/top-level/all-packages.nix

```diff
@@ -10264,6 +10264,8 @@ with pkgs;
 
   touchegg = callPackage ../tools/inputmethods/touchegg { };
 
+  torrent7z = ../tools/archivers/torrent7z { };
```

Comment:
```suggestion
  torrent7z = callPackage ../tools/archivers/torrent7z { };
```

Reply:
oops, big mistake. thanks for mentioning